### DISCUSSION TITLE
Newb looking for feedback

### DISCUSSION THREAD

**Discussion by Nauring** [community/community#52713] [Programming Help]
### Select Topic Area

Product Feedback

### Body

**Introduction:** Hello, I am posting this as feedback request for my small project. (fanpage for game that intrests me)
I am self-thaught, did learn about 30-40h before attempting to jump into this project.
**Goal:** Trying to learn html, css and js(+querry) while doing something fun.
**Limitations:** I have decided against frameworks, php and bootstrap for this first project to not overwhelm myself.

**General help I am looking for:**

    Are there any obvious mistakes I made?
    Is something I done considered no-go ?

**Specyfic things I want to ask about:**

    menu.html is loaded as object. Is this good way to handle leftside menu?
    Some button flicker on page load ? How to fix and why this happens? Does this have something to do with .css and .js linking?
    Assuming no database, is javascript object best way to enter and get data from ?

**Suggestions**
Maybe you have and idea what would be resonable thing for new guy like me to implement as learning experiance on such webiste? 

and here is the link: [MySite](https://nauring.github.io/Test/)

Thanks in advance!

**Comment by tuves** [comment]
Welcome to the GitHub Community, @Nauring, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Reply by MamiStyle** [reply]
hola, donde puedo saber para que sirve esta pagina y como llegue aqui? sospecho que fui manipulada psicologicamente y me hicieron creer cosas que estan fuera de lo real, me ayudaria mucho tu respuesta y agradeceria  tu ayuda
